src/cluster/src/install.rs

```diff
@@ -465,7 +516,7 @@ impl ClusterInstaller {
     ///
     /// ```no_run
     /// use fluvio_cluster::ClusterInstaller;
-    /// let installer = ClusterInstaller::new().build().unwrap();
+    /// let installer = ClusterInstaller::builder().build().unwrap();
```

Comment:
hmm. since this seems not intuitive, having build method on top of build method.

Reply:
This is probably just a problem with the example. There are now two ways to get an installer:

- With all default settings using `ClusterInstaller::new()?`
- With overridden settings using the builder

When using the builder style, you _create_ a builder using `ClusterInstaller::builder()`, and you _finalize_ a builder using `ClusterInstallerBuilder::build()`. So normally you'd have the settings in between those methods, like

```rust
let installer = ClusterInstaller::builder()
    .with_namespace("fluvio")
    .with_rust_log("debug")
    // .with_other_settings()
    .build()?;
```

I'll update the example to make more sense